scripts/spectrum-vars.js

```diff
@@ -138,6 +138,7 @@ const processTypography = async (
 // TODO: use resolve package to find node_modules
 // TODO: we need to revisit whether we need these
 const spectrumPaths = [
+    // Spectrum 1
     path.resolve(
         path.join(
             __dirname,
```

Comment:
Yes, this would definitely be nicer!

My high-level approach with this PR was to apply as light a touch as possible, doing just what was necessary to make the new theme work, but it seems worth doing this to leave things in a cleaner state. I'll go ahead and make the change.

Reply:
Done! I started by applying your suggestion directly, then ended up simplifying a bit more in the process of changing to use `import.meta.resolve` to find the `@spectrum-css` packages.